### ISSUE TITLE
Switching communities does not update team list

### ISSUE THREAD

**Issue by brianlovin** [withspectrum/spectrum#3606]
If you go directly from one community view to another (e.g. via notifications) we don't update the team members list. This is because of the `shouldComponentUpdate` in the `moderatorsList` component - just needs to account for `props.community.id` changing between renders and pass the `scu` check

**Comment by titchimoto** [comment]
Hey @brianlovin are you accepting PR's for this? Would love to have a go at it! :)

Also, I have the project up & running locally, and testing with a couple of different accounts (a twitter one, and a google one...) I don't seem to be actually getting any notifications within the web app...? Not sure I'm missing something? Because I wanted to re-create the issue locally first (I've seen it on the production version)

**Comment by brianlovin** [comment]
@titchimoto yep! You'll need to `yarn run dev:athena` in another terminal window to process the background jobs. This also relies on having redis installed locally on your machine :)

**Comment by titchimoto** [comment]
@brianlovin Ahh awesome, thanks! Was wondering if there was a server/process I missed, d'oh. Thanks, working perfectly now. :)

**Comment by brianlovin** [comment]
@titchimoto your guess sounds about right: probably the cached `moderatorsList` is ending up in the newly queried `communityMembers` data and results in conflicting `user.id` keys.

An obviously solution isn't coming to mind except just forcing the members array through a function to deduplicate users. That's a bit less performant and not ideal, but would solve the problem I suppose. Also: this is getting pretty far into edge-case territory, so I wouldn't spend too much time worrying about this; I think the first fix of switching communities and the `moderatorsList` not updating is a pretty solid fix regardless!

**Comment by titchimoto** [comment]
@brianlovin Haha, that's fair enough. Does seem a little overkill maybe. Will submit the PR for the other fix shortly.
